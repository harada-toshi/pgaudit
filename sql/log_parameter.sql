\! cp sql/log_parameter/log_parameter_default.conf /tmp/config.conf
\! echo "pgaudit.config_file = '/tmp/config.conf'" >> ./tmp_check/data/postgresql.conf
\! rm ./tmp_check/tmp_check.log

\! pg_ctl restart -w -D ./tmp_check/data -l ./tmp_check/tmp_check.log
\! psql contrib_regression -q -a < sql/log_parameter/log_parameter.sql
\! rm /tmp/config.conf

\! cp sql/log_parameter/log_parameter_on.conf /tmp/config.conf
\! pg_ctl restart -w -D ./tmp_check/data -l ./tmp_check/tmp_check.log
\! psql contrib_regression -q -a < sql/log_parameter/log_parameter.sql
\! rm /tmp/config.conf

\! cp sql/log_parameter/log_parameter_off.conf /tmp/config.conf
\! pg_ctl restart -w -D ./tmp_check/data -l ./tmp_check/tmp_check.log
\! psql contrib_regression -q -a < sql/log_parameter/log_parameter.sql
\! rm /tmp/config.conf

\! grep "AUDIT" ./tmp_check/tmp_check.log > results/log_parameter.out

