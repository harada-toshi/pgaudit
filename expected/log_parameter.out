		format='AUDIT: %object_id,%command_parameter,%command_text'
LOG:  AUDIT: ,,BEGIN;
LOG:  AUDIT: ,,PREPARE prep_dx AS SELECT * FROM pg_extension WHERE extname = $1 AND extversion = $2;
LOG:  AUDIT: pg_catalog.pg_extension,,PREPARE prep_dx AS SELECT * FROM pg_extension WHERE extname = $1 AND extversion = $2;
LOG:  AUDIT: ,,EXECUTE prep_dx ('plpgsql','1.0');
LOG:  AUDIT: ,,DEALLOCATE PREPARE prep_dx;
LOG:  AUDIT: ,,COMMIT;
		format='AUDIT: %object_id,%command_parameter,%command_text'
LOG:  AUDIT: ,,BEGIN;
LOG:  AUDIT: ,,PREPARE prep_dx AS SELECT * FROM pg_extension WHERE extname = $1 AND extversion = $2;
LOG:  AUDIT: pg_catalog.pg_extension,plpgsql 1.0,PREPARE prep_dx AS SELECT * FROM pg_extension WHERE extname = $1 AND extversion = $2;
LOG:  AUDIT: ,,EXECUTE prep_dx ('plpgsql','1.0');
LOG:  AUDIT: ,,DEALLOCATE PREPARE prep_dx;
LOG:  AUDIT: ,,COMMIT;
		format='AUDIT: %object_id,%command_parameter,%command_text'
LOG:  AUDIT: ,,BEGIN;
LOG:  AUDIT: ,,PREPARE prep_dx AS SELECT * FROM pg_extension WHERE extname = $1 AND extversion = $2;
LOG:  AUDIT: pg_catalog.pg_extension,,PREPARE prep_dx AS SELECT * FROM pg_extension WHERE extname = $1 AND extversion = $2;
LOG:  AUDIT: ,,EXECUTE prep_dx ('plpgsql','1.0');
LOG:  AUDIT: ,,DEALLOCATE PREPARE prep_dx;
LOG:  AUDIT: ,,COMMIT;
